.changeset/little-news-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@firebase/data-connect': minor
+'firebase': minor
+---
+
+Add custom request headers based on type of SDK calling Core SDK
```

Comment:
This will become release notes shortly. Could you rephrase this with our users in mind. Maybe something like:

> Changed: Add custom request headers based on the type of SDK (TanStack, Angular, etc) that's invoking Data Connect requests. This will help us *something something*.